lambdas/enums/repository_role.py

```diff
@@ -9,4 +9,4 @@ class RepositoryRole(Enum):
 
     @staticmethod
     def list():
-        return [org.value for org in list(RepositoryRole)]
+        return [str(item.value) for item in RepositoryRole]
```

Comment:
Is str here necessary?

Reply:
I've changed the method signature to explicitly return a list of String, and modified a test to be sure, so yes.